internal/application/command.go

```diff
@@ -50,6 +50,8 @@ func GetCommand(ctx context.Context, deviceName string, commandName string, quer
 	_, cmdExist := cache.Profiles().DeviceCommand(device.ProfileName, commandName)
 	if cmdExist {
 		res, err = readDeviceCommand(device, commandName, queryParams, dic)
+	} else if regexCmd {
+		res, err = readDeviceResources(device, commandName, queryParams, dic)
```

Comment:
Had a hard time determining that this was different name than to other function. How about more explicit name.
```suggestion
		res, err = readDeviceResourcesRegex(device, commandName, queryParams, dic)
```